### ISSUE TITLE
Reduce number of stored backups if disk space is low

### ISSUE THREAD

**Issue by willmot** [humanmade/backupwordpress#907]
If the user has low disk space we should try to reduce the number of backups stored to avoid filling things up and ruining their site.

- [ ] Set the default number to keep smartly based on their disk space
- [ ] If disk-space fills up, auto reduce the number stored and email the user to let them know
- [ ] When the user set's the number of backups to keep warn them if the total size of the backups they are storing is going to cause them to run out of disk space.

**Comment by pdewouters** [comment]
Related: https://github.com/humanmade/backupwordpress/issues/615

Would be worth checking with Godaddy if there is a way to reliably retrieve actual allocated disk space for a specific user/

**Comment by willmot** [comment]
> Would be worth checking with Godaddy if there is a way to reliably retrieve actual allocated disk space for a specific user/

Yeah definitely, that would be really useful feedback to provide to them as currently it's not easy for us to respect their disk usage quotas as we can't access them.

**Comment by pdewouters** [comment]
https://garage.godaddy.com/wordpress-plugin-partner-program/directory/

**Comment by willmot** [comment]
Let's apply 👍

Tom
